app/helpers/my_module_tabs_helper.rb

```diff
@@ -0,0 +1,43 @@
+module MyModuleTabsHelper
+  # Options can contain the following elements:
+  # - id - HTML id of the <li> element
+  # - href - If route to the tab is different than
+  #          <tab>_my_module_path(my_module)
+  def my_module_tab_li(my_module, tab, active, glyphicon, opts = {})
+    return '' unless my_module.shown_tabs.include?(tab)
+
+    title = t("nav2.modules.#{tab}")
+    if opts['href'].blank?
+      opts['href'] = send("#{tab}_my_module_path", my_module)
+    end
+
+    res = <<-eos
+    <li id="#{opts['id']}" class="#{'active' if active}">
+      <a href="#{opts['href']}" title="#{title}">
```

Comment:
I think that we should sanitize the title here.
http://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html